spec/services/cfe/obtain_assessment_result_service_spec.rb

```diff
@@ -0,0 +1,143 @@
+require 'rails_helper'
+
+module CFE # rubocop:disable Metrics/ModuleLength
+  RSpec.describe ObtainAssessmentResultService do
+    around do |example|
+      VCR.turn_off!
+      example.run
+      VCR.turn_on!
+    end
+
+    let(:application) { create :legal_aid_application, application_ref: 'L-XYZ-999' }
+    let(:submission) { create :cfe_submission, aasm_state: 'properties_created', legal_aid_application: application }
+    let(:expected_response) { expected_response_hash.to_json }
+    let(:service) { described_class.new(submission) }
+
+    describe '#cfe_url' do
+      it 'contains the submission assessment id' do
+        expect(service.cfe_url)
+          .to eq "#{Rails.configuration.x.check_finanical_eligibility_host}/assessments/#{submission.assessment_id}"
+      end
+    end
+
+    context 'success' do
+      before do
+        stub_request(:get, service.cfe_url)
+          .to_return(body: expected_response)
+      end
+
+      it 'calls the expected URL' do
+        ObtainAssessmentResultService.call(submission)
+      end
```

Comment:
I don't think this is actually testing any expectation.

Reply:
Well, I think my thinking was that it was testing that the expected URL was called, bu that's tested elsewhere, so I've taken it out.